decidim-core/app/validators/etiquette_validator.rb

```diff
@@ -31,12 +30,12 @@ def validate_long_words(record, attribute, value)
   end
 
   def validate_caps_first(record, attribute, value)
-    return if value.scan(/^[a-z]{1}/).empty?
+    return if value.scan(/\A[a-z]{1}/).empty?
     record.errors.add(attribute, options[:message] || :must_start_with_caps)
   end
 
   def validate_length(record, attribute, value)
-    return if value.length > 15
+    return if value.length > 10
```

Comment:
Intended?

Reply:
Yeah, I felt like the previous one was too long. I should have included that in the PR description!

Reply:
Changed my mind and decided not to touch this in this PR!